src/main/java/com/dev/cinema/model/Order.java

```diff
@@ -0,0 +1,80 @@
+package com.dev.cinema.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @OneToMany
+    @JoinColumn(name = "order_id")
+    private List<Ticket> tickets;
+
+    @ManyToOne
```

Comment:
<img width="552" alt="Screenshot 2020-10-12 at 13 47 36" src="https://user-images.githubusercontent.com/68664175/95738193-7fde5300-0c91-11eb-83a3-3fb32eec407d.png">

Reply:
I used FetchType.LAZY in @OneToMany connection, with @ManyToOne we can use  FetchType.EAGER

Reply:
My apologies, you're right